vmdb/spec/lib/extensions/ar_virtual_spec.rb

```diff
@@ -631,47 +631,42 @@ def self.reflections; super.merge(:ref2 => OpenStruct.new(:name => :ref2, :optio
 
     context "virtual column" do
       it "as Symbol" do
-        lambda { Vm.all(:include => :platform) }.should_not raise_error
+        lambda { Vm.includes(:platform).load }.should_not raise_error
```

Comment:
rspec 3.3 will now warn for a bare `raise_error`: https://github.com/rspec/rspec-expectations/pull/768